### DISCUSSION TITLE
[IDEA] Separate Project manager and Notebook theme

### DISCUSSION THREAD

**Discussion by kookma** [NicolasPetton/Projectify#3] [Q&A] [answered]
Hi Nico,
While Project manager and notebook theme in combination create a wonderful tool, but have a look at GG ravels that many use the Notebook theme for different use cases.

I may suggest to publish them separately while Project manager may depend on Notebook theme so on install user will be notify to install Notebook theme too.

**Comment by kookma** [comment, marked as answer]
@NicolasPetton - Oh I downloaded the latest update and I see you have separated them. 
Thank you!

**Reply by NicolasPetton** [reply]
@kookma Indeed, there's no dependency between Notebook and Projectify.

**Reply by kookma** [reply]
Hi Nico,
I download the latest version and I realized Projectify is on its own! Good move!

I hope we could also see the notebook theme repo soon!

**Reply by NicolasPetton** [reply]
Here you go https://github.com/NicolasPetton/Notebook :).

**Reply by kookma** [reply]
Love it!
